site/content/entry/yuzu-progress-report-oct-2022/index.md

```diff
@@ -0,0 +1,314 @@
+However, to help improve yuzu's accuracy to the Switch software, yuzu has been focusing on getting key operating system features working, and one of the most important is multi-process support, since this corresponds to how the Switch software actually works.
+
+byte[] and bunnei have been hard at work this month getting certain prerequisites available for multi-process support. One of those is {{< gh-hovercard "9055" "support for server sessions," >}}  needed for the nx-hbloader homebrew. 
+This is used on the Switch to launch homebrew, and is needed for homebrew that do not work when launched directly from their NRO file.
+
+Additionally, byte[] also needed to {{< gh-hovercard "9137" "fix several services," >}} as the previous stub implementations had assumed that they would never be torn down. 
+With these changes, and the associated graphical fixes, nx-hbloader and nx-hbmenu now finally work as intended! 
+To use them, you can extract hbl.nsp and hbmenu.nro from your Atmosphere installation. 
+Place hbmenu.nro in yuzu's SD card directory, and then launch hbl.nsp, and the homebrew menu will boot:
+
+{{< imgs
+    "./hbmenu.mp4| They launch now"
+  >}}
+
+Let’s talk about amiibos.
+Amiibo data is stored both encrypted and as plain data. 
+To access the encrypted portion, a key dump is required, but not all games use the encrypted portion, some are just fine with the plain text available.
+For those cases, german77 now {{< gh-hovercard "9113" "marks amiibos as read-only if no key dumps are available." >}}
+This won’t replace dumping your amiibo keys if a game needs it, but many games will be fine with just a basic level of read-only detection.
+
+{{< single-title-imgs
+    "Thank you Gidoly and german77 for the pics!"
+    "./amiibo2.png"
+    "./amiibo3.png"
+    "./amiibo1.png"
+    >}}
+
+## Input improvements
+
+Here’s some Switch Vs PC compatibility debate. 
+The emulator performs vibration tests regularly each time the game initializes a controller for vibration, the problem (there’s always a problem) is that some controllers take longer than others to respond.
+Previously, a fixed 15ms delay was implemented, because some controllers just needed that much time to respond.
+If the game performs successive vibration tests, the game is delayed until the controller responds, reducing framerates, for example, from 60 to 32 FPS. 
+All that performance being lost waiting for a bad quality controller.
+
+So, a better middle ground is needed. Thankfully german77 came with a much smarter solution.
+Instead of repeating the test each time, it’s better to {{< gh-hovercard "9107" "cache it the first time and return that same result afterwards." >}}
+A nice free performance boost for generic controller users.
+
+Newcomer [ZwipZwapZapony](https://github.com/ZwipZwapZapony) (love the name) {{< gh-hovercard "9008" "fixed a copy-paste oopsie in how left and right controller colours are identified." >}}
+Thank you!
```

Comment:
```suggestion
So, a better middle ground is needed. Thankfully, german77 came up with a much smarter solution.
Instead of repeating the test each time, it’s better to {{< gh-hovercard "9107" "cache it the first time and return that same result afterwards." >}}
A nice, free, performance boost for those of us who use generic controllers.

Newcomer [ZwipZwapZapony](https://github.com/ZwipZwapZapony) (love the name) {{< gh-hovercard "9008" "fixed a copy-paste oopsie in how left and right controller colours are identified." >}}
Thank you!
```